util/util.go

```diff
@@ -15,7 +18,19 @@ func WriteConfigmap(clientset *kubernetes.Clientset, permission map[string]map[s
 	if err != nil {
 		return err
 	}
+	data := strings.Replace(string(permissions), "\n", "\\n", -1)
```

Comment:
I think it would be more elegant to add the keys 'data' and 'label.yaml' to a map, and then marshal the entire map, instead of using marshaling and string manipulation.

Reply:
i never said its pretty